src/main/java/ca/utoronto/fitbook/adapter/web/UserLoginController.java

```diff
@@ -1,26 +1,28 @@
 package ca.utoronto.fitbook.adapter.web;
 
 import ca.utoronto.fitbook.application.port.in.UserLoginUseCase;
-import ca.utoronto.fitbook.application.port.in.UserRegisterUseCase;
 import ca.utoronto.fitbook.application.port.in.command.UserLoginCommand;
-import ca.utoronto.fitbook.application.port.in.command.UserRegisterCommand;
 import ca.utoronto.fitbook.application.port.out.response.UserLoginResponse;
-import ca.utoronto.fitbook.application.port.out.response.UserRegisterResponse;
 import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 
+import javax.servlet.http.HttpSession;
+
 @Controller
 @RequiredArgsConstructor
 public class UserLoginController {
     private final UserLoginUseCase userLoginUseCase;
 
     @PostMapping(path = "/login")
-    String UserLogin(Model model, @RequestBody UserLoginCommand command) {
+    String loginUser(Model model, HttpSession session, @RequestBody UserLoginCommand command) {
         UserLoginResponse response = userLoginUseCase.loginUser(command);
```

Comment:
Shoudln't it cehck if userId in session is null like above?

Reply:
No, since the session should be blank on login and register.